api/src/router/routes/provision.ts

```diff
@@ -20,12 +20,12 @@
 
 import { asyncMiddleware } from '@bcgov/common-nodejs-utils';
 import express from 'express';
-import { provisionProfileNamespaces, updateProvisionedNamespaces } from '../../controllers/provision';
+import { provisionCallbackHandler, provisionProfileNamespaces } from '../../controllers/provision';
 
 const router = express.Router();
 
 // Provisioning
 router.post('/:profileId/namespace', asyncMiddleware(provisionProfileNamespaces));
-router.put('/namespace', asyncMiddleware(updateProvisionedNamespaces));
+router.put('/namespace', asyncMiddleware(provisionCallbackHandler));
 
-export default router;
+export default router;
```

Comment:
Nit: No newline
```suggestion
export default router;

```

Reply:
The linter we use seems to require a new line for each file - otherwise it would fail the npm run test:lint.